VikingQuiz.Api/Controllers/UsersController.cs

```diff
@@ -58,38 +54,48 @@ public IActionResult Add([FromBody]UserViewModel user)
 
             if (!ModelState.IsValid)
             {
-                return BadRequest(ModelState.Where(x => x.Value.Errors.Count > 0).Select(x => new { x.Key, x.Value.Errors })
-    .ToArray());
+                return BadRequest(ModelState.Where(x => x.Value.Errors.Count > 0).Select(x => new { x.Key, x.Value.Errors }).ToArray());
             }
 
-            User usr = new User {
-                Username = user.Username,
-                Pass = user.Password,
-                Email = user.Email,
-                PictureUrl = user.PictureUrl
-            };
-
-            User newusr = userRepo.CreateUser(usr);
+            User newusr = userRepo.CreateUser(
```

Comment:
Try to avoid abbreviations. It is prefferd to use full name for variables (i.e. newUser, user instead of usr etc.)